.gitlab/internal_image_deploy/internal_image_deploy.yml

```diff
@@ -119,7 +119,12 @@ docker_trigger_cws_instrumentation_internal:
     - source /root/.bashrc
     - export GITLAB_TOKEN=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $GITLAB_SCHEDULER_TOKEN_SSM_NAME)
     - if [ "$BUCKET_BRANCH" = "beta" ] || [ "$BUCKET_BRANCH" = "stable" ]; then TMPL_SRC_REPO="${TMPL_SRC_REPO}-release"; fi
-    - if [ "$BUCKET_BRANCH" = "nightly" ]; then RELEASE_TAG="${RELEASE_TAG}-${CI_COMMIT_SHORT_SHA}"; fi
+    - |
+      if [ "$BUCKET_BRANCH" = "nightly" ]; then
+        RELEASE_TAG="${RELEASE_TAG}-${CI_COMMIT_SHORT_SHA}"
+        TMPL_SRC_REPO="${TMPL_SRC_REPO}-nightly"
+      fi
+    - if [ "$BUCKET_BRANCH" = "dev" ]; then RELEASE_TAG="dev-${RELEASE_TAG}-${CI_COMMIT_SHORT_SHA}"; fi
```

Comment:
IIUC the `dev` uses the default value (those defined as `variables` from line 108) for the `TMPL_SRC_REPO`, whereas the `RELEASE_TAG` default value is for `beta` and `stable`. Could we use all the default values for the same bucket to save a `if` in this script?

Reply:
I'm trying to have the same code as the agent and cluster agent here, I would prefer not diverging so that next time they edit the code they do not forget about us